### PR TITLE
dtbo: allow overriding binary name

### PR DESCRIPTION
**Pull request by sledges**

### CODE REVIEW

File: droid-hal-device.inc [hunk 1]
```diff
@@ -53,6 +53,7 @@
 #                      from droid-hal-device
 # bootloader_binary: define bootloader binary name when aboot is packaged
 #                      separately from droid-hal-device
+# dtbo_binary:       override dtbo binary name. Defaults to dtbo.img
```
Comment (Thaodan) [inline droid-hal-device.inc:56]: I think it should be dtbo_image to fit with the rest.

File: droid-hal-device.inc [hunk 2]
```diff
@@ -95,6 +96,9 @@ such_macro supports spaces\
 %if 0%{!?dhd_path:1}
 %define dhd_path rpm/dhd
 %endif
+%if 0%{!?dtbo_binary:1}
+%define dtbo_binary dtbo.img
```
Comment (Thaodan) [inline droid-hal-device.inc:100]: ^

File: droid-hal-device.inc [hunk 3]
```diff
@@ -813,7 +817,7 @@ fi
 
 touch dtbo.files
 
-if cp out/target/product/%{device}/dtbo.img $RPM_BUILD_ROOT/boot &> /dev/null; then
+if cp out/target/product/%{device}/%{dtbo_binary} $RPM_BUILD_ROOT/boot/dtbo.img &> /dev/null; then
```
Comment (Thaodan) [inline droid-hal-device.inc:820]: ^

**Review by Thaodan — CHANGES_REQUESTED** [review]

File: droid-hal-device.inc (hunk 1 above)
Comment (sledges) [reply droid-hal-device.inc:56]: It's binary, because it fits bootloader_binary, they are both images.

**Comment by Thaodan** [comment]
> > @@ -53,6 +53,7 @@
> 
>  #                      from droid-hal-device
>  # bootloader_binary: define bootloader binary name when aboot is 
packaged
>  #                      separately from droid-hal-device
> +# dtbo_binary:       override dtbo binary name. Defaults to dtbo.img
> 
> It's binary, because it fits bootloader_binary, they are both images.
OK at least AOSP talks about dtbo img.

**Review by jusa — APPROVED** [review]

**Pull request merged**